src/Logger/HistoryLogger.php

```diff
@@ -71,4 +71,29 @@ public function log(HistorizableInterface $entity, string $code, array $data = [
 
         $this->entityManager->flush();
     }
+
+    /**
+     * check and add $updatedEntity diff log between $initalEntity and $updatedEntity
+     */
+    public function logEntityDiff(HistorizableInterface $initalEntity, HistorizableInterface $updatedEntity, string $code, array $data = []): void
+    {
+        $this->logDataDiff(
+            $updatedEntity,
+            $initalEntity->getDataForHistoryDiff(),
+            $updatedEntity->getDataForHistoryDiff(),
+            $code,
+            $data
+        );
+    }
+
+    /**
+     * check and add $entity diff log between $initialData and $updatedData
+     */
+    public function logDataDiff(HistorizableInterface $entity, array $initialData, array $updatedData, string $code, array $data = []): void
```

Comment:
@lfortunier since $entity is the updatedEntity i'm not so sure that you need the $updatedData param